extensions/amp-story/0.1/media-pool.js

```diff
@@ -707,17 +678,20 @@ export class MediaPool {
       return;
     }
 
-    poolMediaEl.pause();
-
-    if (opt_rewindToBeginning) {
-      poolMediaEl.currentTime = 0;
-    }
+    return this.enqueueMediaElementTask_(poolMediaEl, new PauseTask())
+        .then(() => {
+          if (opt_rewindToBeginning) {
+            this.enqueueMediaElementTask_(poolMediaEl, new RewindTask());
```

Comment:
Maybe: ``this.rewindToBeginning(domMediaEl)``

Also, do we actually need to wait for the PauseTask to be completed before we rewind the video? Looks like it was working synchronously before

Reply:
Not sure, this was just to be very cautious.  We can experiment later, since none of this is well-documented :)

Didn't use `rewindToBeginning` to prevent getting the matching element from the pool twice and wasting cycles.